src/hangman/models/game.model.ts

```diff
@@ -8,16 +10,17 @@ import {
   IsUUID,
 } from 'class-validator';
 
-import { NewGameStartedEvent } from '../Events/NewGameStarted.event';
-import { InvalidGameException } from '../../Exceptions';
-import { GameDto } from '../../Infrastructure/Dto/Game.dto';
-import { Field, ObjectType } from '@nestjs/graphql';
-import { LetterGuessedEvent } from '../Events/LetterGuessed.event';
-import { Logger } from '@nestjs/common';
+import { NewGameStartedEvent } from '../events/impl/new-game-started.event';
+import { LetterGuessedEvent } from '../events/impl/letter-guessed.event';
+import { InvalidGameException } from '../exceptions';
+import { GameDto } from '../interfaces/game-dto.interface';
 
 @ObjectType()
 export class Game extends AggregateRoot {
+  @IsUUID()
   public readonly id: string;
+
+  // @IsNumber() // FIXME - caused validation error
```

Comment:
i think because it's not set (in the constructor it's undefined). 
haven't used the version until now. Think it's good to set version to `1` in the constructor, then increase on every event